fuzzing/fuzz.go

```diff
@@ -0,0 +1,26 @@
+package fuzz
```

Comment:
Minor nitpick - please rename this file (and I suppose the output binary) to represent what is being fuzzed (for example, maybe `fuzz_site_check.go`?)